### PR TITLE
inference normalization fix

### PR DESCRIPTION
**Pull request by kartik4949**
Solves #872 
Please test on sample image. @mingxingtan @fsx950223

### CODE REVIEW

**Comment by mingxingtan** [comment]
The image preprocessing is done here: https://github.com/google/automl/blob/9af8a40169b0f5f07c61a3e09ab0077f1f476fdc/efficientdet/keras/efficientdet_keras.py#L894

I don't think you need to normalize images by yourself.

**Comment by kartik4949** [comment]
> The image preprocessing is done here: https://github.com/google/automl/blob/9af8a40169b0f5f07c61a3e09ab0077f1f476fdc/efficientdet/keras/efficientdet_keras.py#L894
> 
> 
> 
> I don't think you need to normalize images by yourself.

Thanks got it
